src/main/kotlin/com/openlattice/assembler/Assembler.kt

```diff
@@ -185,7 +185,7 @@ class Assembler(
 
     @Subscribe
     fun handleEntitySetCreated(entitySetCreatedEvent: EntitySetCreatedEvent) {
-        createOrUpdateProductionViewOfEntitySet(entitySetCreatedEvent.entitySet.id)
+
```

Comment:
remove completely?

Reply:
I left these in on purpose, mainly because all of these will have to get wired up to transporter somehow and we'll end up recreating this logic down the road, but that's what git history is for right?